repo/js/main.js

```
@@ -0,0 +1,232 @@
+                
+                // 方向键
+                "UP": "UP", "上": "UP", "上方向": "UP", "上箭头": "UP",
+                "DOWN": "DOWN", "下": "DOWN", "下方向": "DOWN", "下箭头": "DOWN", 
+                "LEFT": "LEFT", "左": "LEFT", "左方向": "LEFT", "左箭头": "LEFT",
+                "RIGHT": "RIGHT", "右": "RIGHT", "右方向": "RIGHT", "右箭头": "RIGHT",
+                
+                // 修饰键
+                "CTRL": "CONTROL", "CONTROL": "CONTROL", "控制": "CONTROL", "控制键": "CONTROL",
+                "ALT": "MENU", "MENU": "MENU", "菜单": "MENU", "菜单键": "MENU",
+                "SHIFT": "SHIFT", "转换": "SHIFT", "转换键": "SHIFT",
+                "WIN": "LWIN", "窗口": "LWIN", "窗口键": "LWIN",
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)